src/main/java/picard/nio/GoogleStorageUtils.java

```diff
@@ -50,19 +49,20 @@
 class GoogleStorageUtils {
 
     public static void initialize() {
-        // requester pays support is currently not configured
-        CloudStorageFileSystemProvider.setDefaultCloudStorageConfiguration(GoogleStorageUtils.getCloudStorageConfiguration(20, null));
+        final String google_project = System.getProperty("google_project_requester_pays");
```

Comment:
It seems like this is a bit load-order dependent.  Have you checked that the commandline argument overrides the setting in the way you want?  I think it's also possible to get the default google project so it's not necessary to specify at all.  In gatk we instantiate a preferences object and check if it has a value for the project I think, but I'm not sure if it works or has any weirdness.

Reply:
how about we get this simple version to work and if people need something else, it can be added later?